maintainers/maintainer-list.nix

```diff
@@ -6090,6 +6090,11 @@
     githubId = 2147649;
     name = "Euan Kemp";
   };
+  eum3l = {
+    githubId = 77971322;
+    github = "eum3l";
+    name = "Emil";
```

Comment:
It's better to add an email address so others can still contact with you if anything happens to your GitHub account.